rfcs/0000-dns-cluster-aware-delegation.md

```diff
@@ -34,33 +34,36 @@ The mode will determine the behaviour of the reconciliation of all DNSRecords th
 ### Default (current behaviour)
 
 DNSRecords will be reconciled and published to the provider by this instance with no delegation, this is the same behaviour as the operator today and is the default mode used if none is specified.
-DNSRecords on these clusters must have access to valid provider credentials referenced in the spec.
+
 This mode should be selected for single cluster scenarios, and/or where the current approach to multi cluster is to be maintained.
 
 ### Primary
 
-DNSRecords will be reconciled and published to the provider by this instance through an authoritative DNSRecord which is created on behalf of the primary DNSRecord that all publishing requests are delegated to.
-DNSRecords on these clusters must have access to valid provider credentials referenced in the spec.
-The authoritative DNSRecord is created in the same namespace as the original DNSRecord (created by the DNSPolicy or otherwise) and at a minimum will contain the primary DNSRecords endpoints.
-The remaining specification of the authoritative DNSRecord is synced from the original primary DNSRecord (rootHost, providerRef etc..) 
-This mode should be selected for multi cluster scenarios where a designated "control plane" cluster(s) are being configured with elevated privileges to access external DNS providers.
+DNSRecords will be reconciled and published to the provider through an authoritative zone DNSRecord which is created on behalf of the primary DNSRecord with all publishing requests delegated to it.
+The dns operator will ensure the existence of the authoritative zone DNSRecord, with:
+  * the same `metdata.namespace` as the current record
+  * a `metadata.name` generated based on the current records `spec.rootHost` i.e. `foo-example-com_authoritative_zone`
```

Comment:
nit: I know just an example but prob would keep the suffix as short as possible as there is a max length to consider